<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OverviewPage</class>
 <widget class="QWidget" name="OverviewPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>629</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>149</y>
     <width>861</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>10</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QWidget" name="ui_transactions" native="true">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background:'#102537';
margin-top:10px;</string>
        </property>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color:&quot;#0adbf0&quot;;font-weight:bold;</string>
         </property>
         <property name="text">
          <string>SHARD/USD</string>
         </property>
        </widget>
        <widget class="QLabel" name="m_price">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>70</y>
           <width>181</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color:WHITE</string>
         </property>
         <property name="text">
          <string>$ 0.00 / SHARD</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QComboBox" name="changeTime">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>10</y>
           <width>101</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QComboBox#changeTime {
    background:&quot;#0b1d2c&quot;;
    border:0px;
    color:white;
    outline: none;
    padding-left:5px;
   }
   QComboBox::drop-down {
    
        border:0px;
        margin-right:10px;

    }
    QComboBox::down-arrow {
        image: url(&quot;:/images/res/images/dropdown.png&quot;);
    }
    QComboBox#changeTime:pressed {
        background: &quot;#0b1d2c&quot;; 
    }
    QComboBox#changeTime:focus:pressed {
        background: &quot;#003750&quot;;
     }
    QComboBox#changeTime:focus { 
        background: &quot;#0b1d2c&quot;; 
    }
    QComboBox#changeTime:hover {
         background: &quot;#003750&quot;;
    }

    QComboBox#changeTime:checked { 
       background: &quot;#0b1d2c&quot;;
    }
    QComboBox#changeTime:checked:hover {
        background: &quot;#002d3&quot;;&quot;
 }
QComboBox#changeTime:disabled { 
    background: &quot;#0a2634&quot;;
	color:gray;
 }
    </string>
         </property>
         <property name="currentText">
          <string/>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
        <widget class="QComboBox" name="graphSelector">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>10</y>
           <width>161</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QComboBox {
    background:&quot;#0a2634&quot;;
    border:0px;
    color:white;
    outline: none;
    padding-left:5px;
	font-size:20px;
   }
   QComboBox::drop-down {
    
        border:0px;
        margin-right:10px;

    }
    QComboBox::down-arrow {
        image: url(&quot;:/images/res/images/dropdown.png&quot;);
    }
    QComboBox:pressed {
        background: &quot;#0b1d2c&quot;; 
    }
    QComboBox:focus:pressed {
        background: &quot;#003750&quot;;
     }
    QComboBox#changeTime:focus { 
        background: &quot;#0b1d2c&quot;; 
    }
    QComboBox:hover {
         background: &quot;#003750&quot;;
    }

    QComboBox:checked { 
       background: &quot;#0b1d2c&quot;;
    }
    QComboBox:checked:hover {
        background: &quot;#002d3&quot;;
 }
QComboBox:disabled { 
    background: &quot;#0a2634&quot;;
	color:gray;
 }
    </string>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>41</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;img src=&quot;:/images/res/images/graph.png&quot;&gt;</string>
         </property>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>401</width>
           <height>371</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QCustomPlot" name="graphView" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_8"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="styleSheet">
                 <string notr="true">color:gray</string>
                </property>
                <property name="text">
                 <string>MCAP:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="m_cap">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">color:white;margin-top:-4px;</string>
                </property>
                <property name="text">
                 <string>0.00 $</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="styleSheet">
                 <string notr="true">color:gray</string>
                </property>
                <property name="text">
                 <string>Change (24h):</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="m_change">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">color:white;margin-top:-4px;</string>
                </property>
                <property name="text">
                 <string>0.00 $</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_10">
              <item>
               <widget class="QLabel" name="label_10">
                <property name="styleSheet">
                 <string notr="true">color:gray</string>
                </property>
                <property name="text">
                 <string>Volume (24h):</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="m_volume">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">color:white;margin-top:-4px;</string>
                </property>
                <property name="text">
                 <string>0.00 $</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="Line" name="line">
         <property name="geometry">
          <rect>
           <x>-10</x>
           <y>56</y>
           <width>441</width>
           <height>20</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color:'#0b1d2c';</string>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QWidget" name="ui_graph" native="true">
        <property name="styleSheet">
         <string notr="true">background:'#102537';
margin-top:10px;</string>
        </property>
        <widget class="ListTransactions" name="listTransactions">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>60</y>
           <width>412</width>
           <height>411</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">padding:5px;border:0px</string>
         </property>
        </widget>
        <widget class="QPushButton" name="moreButton">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>10</y>
           <width>81</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton { 
 background-color: &quot;#0b1d2c&quot;;
	border:1px solid gray;
	color:white;
	outline: none;

}

QPushButton:pressed {
   background-color: &quot;#0b1d2c&quot;;
border:1px solid gray;
	color:white;

}
QPushButton:focus:pressed { 
    background: &quot;#003750&quot;;
	border:1px solid #0adbf0;
	color:white;

 }
QPushButton:focus { 
   background-color: &quot;#0b1d2c&quot;;
	border:1px solid gray;
	color:white;

}
QPushButton:hover {
     background-color: &quot;#0b1d2c&quot;;
	border:1px solid #0adbf0;
 }
QPushButton:checked { 
     background-color: &quot;#0b1d2c&quot;;
border:1px solid gray;
	color:white;

 }
QPushButton:disabled { 
    background:'#0a2634';
color:gray;
border:1px solid gray;
 }
</string>
         </property>
         <property name="text">
          <string>MORE</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>10</y>
           <width>221</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color:white;
font-size:18px;</string>
         </property>
         <property name="text">
          <string>&lt;img src=&quot;:/images/res/images/transactions.png&quot;&gt;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;List Transactions</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-1</y>
     <width>861</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background:'#102537';border:0px</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>131</width>
         <height>151</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background:'#0e2232';border:0px</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QPushButton" name="ui_recieve">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>131</width>
          <height>131</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">border:0px;
background-image:url(:/images/res/images/recieve.png);
background-repeat:no-repeat;
background-position: center;
background-attachment:fixed;
background-size:cover;
outline: none;
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>110</y>
          <width>131</width>
          <height>41</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">margin-top:5px;
margin-bottom:20px;
color:white;
border:0px</string>
        </property>
        <property name="text">
         <string>Recieve</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>0</y>
         <width>141</width>
         <height>141</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:'#102537';border:0px</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;
font-size:10px</string>
        </property>
        <property name="text">
         <string>UNCONFIRMED</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="labelUnconfirmed">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>60</y>
          <width>131</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>730</x>
         <y>0</y>
         <width>131</width>
         <height>151</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:'#0e2232';border:0px</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QPushButton" name="ui_send">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>131</width>
          <height>131</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">border:0px;
background-image:url(:/images/res/images/send.png);
background-repeat:no-repeat;
background-position: center;
outline:none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>110</y>
          <width>131</width>
          <height>41</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">margin-top:5px;
margin-bottom:20px;
color:white;
border:0px</string>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>0</y>
         <width>321</width>
         <height>141</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:'#102537';border:0px</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>321</width>
          <height>20</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;</string>
        </property>
        <property name="text">
         <string>BALANCE</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="labelBalance">
        <property name="geometry">
         <rect>
          <x>6</x>
          <y>50</y>
          <width>311</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>0</y>
         <width>131</width>
         <height>141</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:'#102537';border:0px</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;
font-size:10px</string>
        </property>
        <property name="text">
         <string>STAKED</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="labelStake">
        <property name="geometry">
         <rect>
          <x>16</x>
          <y>60</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ListTransactions</class>
   <extends>QListView</extends>
   <header>listtransactions.h</header>
  </customwidget>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>moreButton</tabstop>
  <tabstop>listTransactions</tabstop>
  <tabstop>ui_recieve</tabstop>
  <tabstop>ui_send</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
